API.yaml

```diff
@@ -481,11 +481,15 @@ paths:
       requestBody:
         description: Query parameters to fetch the timegraph states. The object 'requested_timerange'
           is the requested time range and number of samples. The array 'requested_items'
-          is the list of entryId being requested.
+          is the list of entryId being requested. The boolean optional 'is_full_search' if is true
+          then the full time range between the first and last requested timestamp should be searched
+          for filter matches. The map 'filter_expressions_map' where the key 1 is DIMMED and 4 is EXCLUDED,
```

Comment:
same as above

Reply:
Hi MatthewKhouzam, 

I did following bhufmann's recommendation above 'Map<Integer, Collection<String>> filterExpressionsMap;'. 
As I understood, the frontend is supposed to rematch based on 'DIMMED = 1 << 0' or 'EXCLUDE = 1 << 2'